export const verifiedUser = async (req, res, next) => {
    try {
        
    } catch (error) {
        
    }
}